### PR TITLE
Migration: Admin orgs list

### PR DESCRIPTION
**Pull request by tskarhed**
Angular and Forms migration
#22862
#21484

### CODE REVIEW

File: public/app/features/admin/AdminListOrgsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  return await getBackendSrv().get('/api/orgs');
+};
+
+export const AdminListOrgsPage: FC = () => {
+  const navIndex = useSelector((state: StoreState) => state.navIndex);
+  const navModel = getNavModel(navIndex, 'global-orgs');
+
+  const [deleteOrg, setDeleteOrg] = useState<Organization>(null);
+  const [orgs, setOrgs] = useState<Organization[]>([]);
+  const [, fetchOrgs] = useAsyncFn(getOrgs, []);
```
Comment (Clarity-89) [inline public/app/features/admin/AdminListOrgsPage.tsx:20]: can also do `const fetchOrgs = useAsyncFn(getOrgs, [])[1];`.

File: public/app/routes/routes.ts [hunk 2]
```diff
@@ -344,9 +344,11 @@ export function setupAngularRoutes($routeProvider: route.IRouteProvider, $locati
...
     })
     .when('/admin/orgs', {
-      templateUrl: 'public/app/features/admin/partials/orgs.html',
-      controller: 'AdminListOrgsCtrl',
-      controllerAs: 'ctrl',
+      template: '<react-container />',
+      resolve: {
+        component: () =>
+          SafeDynamicImport(import(/* webpackChunkName: "AdminListOrgsPage" */ 'app/features/admin/AdminListOrgsPage')),
+      },
```
Comment (Clarity-89) [inline public/app/routes/routes.ts:351]: Might also need to add `reloadOnSearch: false` so it doesn't re-render on url query params changes (e.g. when you open/close dashboard search).

Comment (tskarhed) [reply public/app/routes/routes.ts:351]: Is this something that affects more views?

Comment (Clarity-89) [reply public/app/routes/routes.ts:351]: Mostly the views that fetch some data from API when they render.

Comment (tskarhed) [reply public/app/routes/routes.ts:351]: Ah, so basically all migrations I did today 😅

**Pull request closed without merging**